lib/semantics/resolve-names-utils.cc

```diff
@@ -82,6 +82,25 @@ bool IsLogicalConstant(
           (str == ".t" || str == ".f."));
 }
 
+// The operators <, <=, >, >=, ==, and /= always have the same interpretations
+// as the operators .LT., .LE., .GT., .GE., .EQ., and .NE., respectively.
+std::optional<std::string> GenericSpecInfo::GetOtherName() const {
+  std::string name{symbolName_.value().ToString()};
```

Comment:
```name``` and ```otherName``` could be declared ```const```.

Reply:
We don't normally declare local variables `const`.